index(port).html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Home</title>
+    <link href="normalize.css" rel="stylesheet">
+    <link href="styles/styles(port).css" rel="stylesheet">
+  </head>
+  <body class="container">
+
+    <section class="header">
+      <li><a href="portfolio.html">portfolio.</a></li>
```

Comment:
Also `li` can't be a child of `section`, it can only be a child element of `ul` or `ol`.